package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "1.18.0",
+  "version": "1.18.1",
```

Comment:
please use `npm version patch` so that both package.json and package-lock.json get updated with the new version

Reply:
ahh thanks!